### PR TITLE
[docs] Added a contribution guide, code of conduct and issue template (closes #703)

### LINKED ISSUE #703: We need a contribution guide, issue and bug report templates

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @DevExpress/testcafe-docs 

Not checked yet so just take a look.

fixes #703

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+### How would you reproduce this behavior (if this is a bug)?
+
+
+
+### What is the expected behavior?
+
+
+
+### Provide the test code and the tested page URL (if applicable)
```
Comment (inikulin) [inline .github/ISSUE_TEMPLATE.md]: It should be subsection of `How would you reproduce this behavior (if this is a bug)?`

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+### What is the current behavior?
+
+
+
+### How would you reproduce this behavior (if this is a bug)?
+
+
+
+### What is the expected behavior?
```
Comment (inikulin) [inline .github/ISSUE_TEMPLATE.md]: We need question about actual behavior

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at ivann@devexpress.com. All
```
Comment (inikulin) [inline CODE_OF_CONDUCT.md]: `ivann@devexpress.com` is not my email. Use `ivannikulin@devexpress.com` or even better `ifaaan@gmail.com`

Comment (VasilyStrelyaev) [inline CODE_OF_CONDUCT.md]: Argh, I copy-pasted from Mehul's email =)

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (VasilyStrelyaev) [inline .github/ISSUE_TEMPLATE.md]: Don't we have it?

https://github.com/DevExpress/testcafe/pull/868/files#diff-1a1c3dd142f76a5fad803a0c52839881R5

Comment (inikulin) [inline .github/ISSUE_TEMPLATE.md]: We need them to go one after another then.

Comment (VasilyStrelyaev) [inline .github/ISSUE_TEMPLATE.md]: ah, okay

**Pull request merged**